Gemfile.lock

```diff
@@ -27,47 +27,31 @@ PATH
 GEM
```

Comment:
I'm open to it being either way, so whichever way you decide to go with is fine with me.

I suspect the reason some plugins adopted this is that you can't pin the dependencies for all vagrant plugins with vagrant, so the standard behaviour of installing plugins without a Gemfile.lock will pull in the latest gems allowed at install time. Looking at other plugins I'm using it does seem to depend on the project.

With Gemfile.lock
vagrant-lxc
vagrant-cachier
vagrant-chef-zero

Without:
vagrant-hostmanager
vagrant-libvirt
vagrant-mutate
vagrant-omnibus
vagrant-proxyconf